packages/chains/src/chainId.ts

```diff
@@ -17,3 +17,15 @@ export enum ChainId {
   BASE = 8453,
   BASE_TESTNET = 84531,
 }
+
+export enum TestnetChainId {
```

Comment:
Let's keep the single source of truth. It would be a bit annoying to maintain the chain id in two enums. How about we add 
```typescript
export const testnets = [ChainId.BSC_TESTNET, ...] as const
```
or mainnets vise versa